src/SFA.DAS.FindEmploymentSchemes.UnitTests/Web/Services/FilterServiceTests.cs

```diff
@@ -17,14 +21,15 @@ public class FilterServiceTests
         [ClassData(typeof(FilterServiceTestData))]
         public void ApplyFilters_Result(IEnumerable<Scheme> expectedSchemes, SchemeFilterViewModel filters)
         {
-            IFilterService service = A.Fake<IFilterService>();
+            IContentService contentService = A.Fake<IContentService>();
+            ISchemesModelService schemesModelService = A.Fake<ISchemesModelService>();
+
+            FilterService service = new FilterService(contentService, schemesModelService);
             HomeModel model = A.Fake<HomeModel>(x => x.WithArgumentsForConstructor(() => new HomeModel(null, expectedSchemes, null, false)));
-            A.CallTo(() => service.ApplyFilter(filters)).Returns(model);
```

Comment:
The code under test shouldn't be mocked, so we should remove this.

Reply:
Addressed in latest commit